assignments/assignment7/faulty-oops.md

```diff
@@ -0,0 +1,53 @@
+Unable to handle kernel NULL pointer dereference at virtual address 0000000000000000
```

Comment:
@cchoi22915 use markdown, see how this renders without it? https://github.com/cu-ecen-aeld/assignments-3-and-later-cchoi22915/blob/08e570ee11426e232270a0dec8b528ee482773b9/assignments/assignment7/faulty-oops.md